homeassistant/components/worldclock/sensor.py

```diff
@@ -4,7 +4,7 @@
 import voluptuous as vol
 
 from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import CONF_NAME, CONF_TIME_ZONE
+from homeassistant.const import CONF_NAME, CONF_TIME_ZONE, CONF_TIME_FORMAT
```

Comment:
Thanks! Added to the homeassistant.const file.

Reply:
`const.py` is the wrong place. `CONF_TIME_FORMAT` should be defined here only as it's not used by any other integration.

Reply:
Valid. Would it make sense to leave it in const.py because of future additions to other components? That way users don't have to create value templates every time they wish to add a customized view to a clock